react/ReadMore/ReadMore.demo.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import ReadMore from './ReadMore';
+import Text from '../Text/Text';
+import Card from '../Card/Card';
+import Section from '../Section/Section';
+
+const longText = (
+  <Text raw>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus vitae nulla
+    venenatis purus placerat sodales. Aenean ipsum metus, imperdiet eu iaculis
+    at, sollicitudin sed felis. Mauris faucibus risus at ex facilisis volutpat.
+    Integer volutpat ullamcorper velit at mattis. Cras ultrices vestibulum
+    egestas. Vivamus sollicitudin metus nec nulla ullamcorper ultrices vitae in
+    est. Maecenas scelerisque turpis ac ultrices convallis. Integer maximus
+    libero augue, sit amet maximus risus interdum in. Praesent congue feugiat
+    consequat. Suspendisse feugiat mauris non dui aliquet suscipit.
+  </Text>
+);
+
+const mediumText = (
+  <Text raw>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus vitae nulla
+    venenatis purus placerat sodales. Aenean ipsum metus, imperdiet eu iaculis
+    at, sollicitudin sed felis. Mauris faucibus risus at ex facilisis volutpat.
+    Integer volutpat ullamcorper velit.
+  </Text>
+);
+
+const shortText = (
+  <Text raw>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus vitae nulla
+    venenatis purus placerat sodales.
+  </Text>
+);
+
+const ReadMoreContainer = ({ component: DemoComponent, componentProps }) => (
+  <Card style={{ width: 500 }}>
+    <Section>
+      <DemoComponent {...componentProps} />
+    </Section>
+  </Card>
+);
+
+ReadMoreContainer.propTypes = {
+  component: PropTypes.any,
+  componentProps: PropTypes.object.isRequired
+};
+
+export default {
+  route: '/readmore',
+  title: 'ReadMore',
```

Comment:
For consistency, should be `route: '/read-more', title: 'Read More'`.

Sorry, this should probably be automated somehow.